tac/agents/v1/base/game_instance.py

```diff
@@ -85,6 +87,7 @@ def __init__(self, agent_name: str,
 
         :param agent_name: the name of the agent.
         :param strategy: the strategy of the agent.
+        :param mail_starts: the mail stats of the mailbox.
```

Comment:
typo: `mail_starts` instead of `mail_stats`

Reply:
Fixed, thanks!